examples/lighting-app/nrfconnect/prj_sleepy.conf

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
I think we would prefer not adding another variant to this example. The reason is that we try to internally (in Nordic) test and take responsibility for all the available variants, and testing all Thread device types for all examples would cost a lot of CI time. Therefore, we try to make different examples different Thread device types - window covering is SSED, lock is SED, lighting is Router etc. That way we cover different scenarios without combinatorial explosion of tested variants. Can you describe your needs - will you be using this example as SSED in a longer period of time? Maybe we could change the configuration so that you can build the SSED variant just by passing `-DCONFIG_CHIP_THREAD_SSED=y` flag (and, optionally, SED intervals) to `west build` command?

Reply:
Hi Damian, yes I think that's fine. We can build using a flag like`-DCONFIG_CHIP_THREAD_SSED=y`. I'll leave the change to `AppTask.cpp` in this PR to support that use case

Reply:
@andy31415 this change became just a small fix to allow the user to pass a CSL build option on lighting app. I'll leave to Nordic's discretion if they want to add that to Readme, as this could imply responsibility for running a CI for that specific build configuration.